src/qweb/extensions.ts

```diff
@@ -51,13 +51,15 @@ QWeb.addDirective({
     let params = extraArgs ? `owner, ${ctx.formatExpression(extraArgs)}` : "owner";
     let handler;
     if (mods.length > 0) {
+      const handlerKey = `handler${ctx.generateID()}`;
+      ctx.addLine(`const ${handlerKey} = context['${handlerName}'].bind(${params});`);
       handler = `function (e) {`;
       handler += mods
         .map(function(mod) {
           return MODS_CODE[mod];
         })
         .join("");
-      handler += `context['${handlerName}'].call(${params}, e);}`;
+      handler += `${handlerKey}.call(owner, e);}`;
```

Comment:
if it is already bound, maybe you can simply do that: 
```suggestion
      handler += `${handlerKey}(e);}`;
```
not sure if it works though